R/package.R

```diff
@@ -227,9 +227,41 @@ do_check <- function(
   }
 
   # user supplied env vars take precedence
-  if (length(env)) chkenv[names(env)] <- env
+  if (length(env)) {
+    chkenv[names(env)] <- env
+  }
+
+  if (!quiet) {
+    cat_head("R CMD check")
```

Comment:
I think you could drop this

Reply:
I think this is from the original implementation, I just threw everything else into the same if (!quiet) block https://github.com/r-lib/rcmdcheck/blob/224457693c37d655e3c8541a5026aad5ec52ca95/R/package.R#L232

Do you want it removed?